scripts/new.js

```diff
@@ -0,0 +1,47 @@
+import fs from "node:fs";
+
+const episode = Number(process.env.npm_config_episode);
+
+if (!episode) {
+	console.error(
+		"Please, set the episode variable. For example, `npm run new --episode=417`"
```

Comment:
```suggestion
		"Нужен номер эпизода, например: npm run new --episode=417"
```

Пусть говорит со мной на одном языке)